src/app/screens/accountList/index.tsx

```diff
@@ -118,33 +92,36 @@ function AccountList(): JSX.Element {
 
   return (
     <Container>
-      <TopRow title={t('CHANGE_ACCOUNT')} onClick={handleBackButtonClick} />
-      <AccountContainer>
-        {displayedAccountsList.map((account) => (
-          <div key={account.btcAddress}>
-            <AccountRow
-              account={account}
-              isSelected={isAccountSelected(account)}
-              onAccountSelected={handleAccountSelect}
-              isAccountListView
-            />
-            <Separator />
-          </div>
-        ))}
-      </AccountContainer>
+      <div>
+        <TopRow title="" onClick={handleBackButtonClick} />
```

Comment:
Could we consider modifying `TopRow` to make the `title` prop optional?

Reply:
Makes sense to me, I'll make this change